db/migrate/20230106114221_update_default_cohort_for_ehco.rb

```diff
@@ -1,6 +1,6 @@
 class UpdateDefaultCohortForEhco < ActiveRecord::Migration[6.1]
   def change
-    ehco_course = Course.find_by_code(code: :EHCO)
+    ehco_course = Course.find_by(identifier: "npq-early-headship-coaching-offer")
```

Comment:
This has already been run, but find_by_code has been removed so I had to update this.